### PR TITLE
fix(openapi): Fix more empty arrays

### PR DESCRIPTION
**Pull request by nickvergessen**
### ☑️ Resolves

* Next step for https://github.com/nextcloud/spreed/pull/13792

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not possible
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

**Comment by nickvergessen** [comment]
The type on the capabilities changed (from `array{spreed: array{…}}|array<empty>` to `array{spreed?: …}`
And now Jest is failing. Could you have a look @DorraJaouad ?

**Review by miaulalala — APPROVED** [review]
LGTM

**Comment by DorraJaouad** [comment]
> The type on the capabilities changed (from `array{spreed: array{…}}|array<empty>` to `array{spreed?: …}` And now Jest is failing. Could you have a look @DorraJaouad ?

Sorry, I missed the notification for this 👀

File: src/services/CapabilitiesManager.ts [hunk 1]
```diff
@@ -125,12 +125,12 @@ export async function setRemoteCapabilities(joinRoomResponse: JoinRoomFullRespon
 	talkHashStore.setTalkProxyHashDirty(token)
 
 	const response = await getRemoteCapabilities(token)
-	if (Array.isArray(response.data.ocs.data)) {
-		// unknown[] received from server, nothing to update with
+	if (!Object.keys(response.data.ocs.data).length) {
```
[the remark this answers is not in the document]
Comment (Antreesy) [reply src/services/CapabilitiesManager.ts:128]: No. Lickily, `Array` is an object
image

**Pull request merged**